backend/README.md

```diff
@@ -116,28 +113,26 @@ Stamp current version of database to last (useful for migration):
 Running Tracim Backend WSGI APP
 ---------------
 
-You can run Tracim wsgi apps with many wsgi server. We provided here example to run them:
-- with UWSGI using wsgi/* script.
-- with pserve command of pyramid which rely only on development.ini pastedeploy config.
+You can run Tracim WSGI apps with many WSGI servers. We provide examples to run them with:
+- uWSGI using wsgi/* script.
+- The pserve command of pyramid which only rely on development.ini pastedeploy config.
 
-### With Uwsgi : great for production ###
+### With uWSGI: great for production ###
 
-#### Install Uwsgi
+#### Install uWSGI
 
-You can either install uwsgi with pip or with you distrib package manager:
+On Debian:
 
-    # install uwsgi with pip ( unneeded if you already have uwsgi with python3 plugin enabled)
-    sudo pip3 install uwsgi
+    sudo apt install uwsgi uwsgi-plugin-python3
 
-or on debian 9 :
+Or with PIP:
 
-    # install uwsgi on debian 9
-    sudo apt install uwsgi uwsgi-plugin-python3
+    sudo pip3 install uwsgi
 
 #### All in terminal way ####
 
 
-Run all web services with uwsgi
+Run all web services with UWSGI:
```

Comment:
@Skylsmoi @raphj the project uwsgi is uWSGI, not UWSGI

Reply:
Indeed, I messed up this occurrence. Sorry for that. I pushed a commit to my branch to fix this, don't hesitate to merge it. I can also open a new pull request and I don't mind if you fix this without using my commit.

Here is the patch file if it is more convenient for you (`git am < 0001-Fix-UWSGI-uWSGI-in-the-backend-installation-doc.patch`). GitHub does not let me upload the patch as a file.

```
From 137db6107400b62cbe685027ebe2b1809b79eb19 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Rapha=C3=ABl=20Jakse?= <raphael.github@jakse.fr>
Date: Fri, 10 Jan 2020 08:27:28 +0100
Subject: [PATCH] =?UTF-8?q?Fix=20UWSGI=20=E2=86=92=20uWSGI=20in=20the=20ba?=
 =?UTF-8?q?ckend=20installation=20doc?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Raphaël Jakse <raphael.github@jakse.fr>
---
 backend/README.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/backend/README.md b/backend/README.md
index 9e86056d..2a6ed3c0 100644
--- a/backend/README.md
+++ b/backend/README.md
@@ -132,7 +132,7 @@ Or with PIP:
 #### All in terminal way ####
 
 
-Run all web services with UWSGI:
+Run all web services with uWSGI:
 
     ## UWSGI SERVICES
     # set tracim_conf_file path
-- 
2.24.1
```